public/app/plugins/datasource/graphite/datasource.ts

```diff
@@ -117,6 +120,20 @@ export class GraphiteDatasource {
     return tags;
   }
 
+  interpolateVariablesInQueries(queries: GraphiteQuery[]): GraphiteQuery[] {
+    let expandedQueries = queries;
+    if (queries && queries.length > 0) {
+      expandedQueries = queries.map(query => {
+        const expandedQuery = {
+          ...query,
+          target: this.templateSrv.replace(query.target),
+        };
+        return expandedQuery;
+      });
+    }
+    return expandedQueries;
+  }
+
```

Comment:
@torkelo does this look good for function, that basically just interpolates variables and then it is used in explore. 🙂

Reply:
yes, looks ok!